timeseries/src/autogluon/timeseries/models/chronos/pipeline/chronos_bolt.py

```diff
@@ -375,6 +381,9 @@ class ChronosBoltPipeline(BaseChronosPipeline):
     def __init__(self, model: ChronosBoltModelForForecasting):
         self.model = model
 
+        # for easy access to the inner HF-style model
+        self.inner_model = model
```

Comment:
I like this, however in this case we should maybe get rid of `self.model` to avoid confusion? Also, I would optionally add somehow add this, maybe as an instance variable to `BaseChronosPipeline.__init__` to hint subclasses will implement this.

Reply:
Responded above.

Reply:
Done.